src/config.rs

```diff
@@ -32,6 +33,20 @@ pub struct SubsetConfig {
     pub base_url: Option<String>,
 }
 
+/// A command to run during the build/publish process. Given as an array of arguments, with the first argument being
+/// the path to the program. Arguments are not processed by a shell; they are passed directly to the program.
+#[derive(Deserialize, Debug, Default, Clone)]
+pub struct ConfigHook(Option<Vec<String>>);
```

Comment:
I'd be tempted to promote this `Option<>` up to the `ConfigHooks` struct instead.  It feels odd to me for it to be here.